src/FNA3D_Driver_Vulkan.c

```diff
@@ -11264,7 +11163,6 @@ static void VULKAN_SetVertexBufferData(
 	int32_t vertexStride,
 	FNA3D_SetDataOptions options
 ) {
-	/* FIXME: use staging buffer for elementSizeInBytes < vertexStride */
```

Comment:
This FIXME is actually related to a weird thing in XNA where SetData sends data that's meant to be copied in segments rather than as a continuous block of memory, so it should be kept.